CommunicationLayer/CommDeviceControl/RadioCommDevice.cpp

```diff
@@ -0,0 +1,55 @@
+#include <QDebug>
+#include <QTimer>
+#include "RadioCommDevice.h"
+
+namespace
+{
+   const QString DEFAULT_PORT_NAME = "/dev/pts/7";
```

Comment:
In the future I would suggest putting an ifdef around this for both testing an production.